packages/mdc-notched-outline/mdc-notched-outline.scss

```diff
@@ -69,13 +70,20 @@
 
     &__notch {
       width: auto;
+      max-width: 100%;
     }
 
     .mdc-floating-label {
       display: inline-block;
       position: relative;
       top: 17px;
       bottom: auto;
+      max-width: 100%;
+    }
+
+    .mdc-floating-label--float-above {
+      max-width: calc(100% / .75);
```

Comment:
This seems incorrect and seems to be causing a FOUC. If I comment out the text field instantiation in fixture.js, the floated labels are the wrong width. If I set this to simply 100%, they're the correct length.

Reply:
I moved this so it only applied when the `--upgraded` class is added.